test/rest_api/test_hinted_handoff.py

```diff
@@ -0,0 +1,21 @@
+import requests
+import urllib.parse
+import base64
+
+def test_hinted_handoff_sync_point(rest_api):
```

Comment:
Please change the test name to something more descriptive. It's true that it tests the hinted handoff sync point feature, but this is not the only test case that you could come up with for it. More specifically, this test is for the checksum functionality that you introduced and it verifies that the checksum does detect invalid sync points and returns a proper error. For example, you could change it to `test_sync_point_checksum` (the fact that this is for _hinted handoff_ is implied by the file name).